### ISSUE TITLE
How to start the server?

### ISSUE THREAD

**Issue by darrylgibbs** [assetto-corsa-web/accweb#105]
I'm sorry but I must be a real idiot... in the README.md, MANUAL INSTALLATION, #6 says "start accweb using ./accweb on Linux and accweb.exe on Windows".  I'm using Linux, but I cannot find either of these 2 mentioned files in this .zip I downloaded.

I literally downloaded it, took the sample_config.yml, editted out the SSL stuff (just for now), placed in my own passwords and now I'm stuck.  What am I missing??

**Comment by Kugelschieber** [comment]
You need to run the `accweb` binary in the main folder (about 6 MB). The dot just indicates that you need to run it inside the folder.

```
cd /your/accweb/installation/directory
./accweb
```

If it does not start, run `chmod +x accweb` inside the folder.

**Comment by darrylgibbs** [comment]
The entire size of the .zip is only 0.239mb? And those files just aren't there. I've added screenshots of the zip and the extracted files on my server.  The only thing I've added is my accServer.exe file.

![Screenshot 2020-10-11 093008](https://user-images.githubusercontent.com/9167471/95678664-6e6b4d00-0ba4-11eb-8a2b-b0c6169ba771.png)
![Screenshot 2020-10-11 093032](https://user-images.githubusercontent.com/9167471/95678665-6f9c7a00-0ba4-11eb-9d12-38f1a15996db.png)
